docs/sdks/client-sdks/react-native/initialization.mdx

```diff
@@ -0,0 +1,305 @@
+You could consider caches always expired so that it non-blocking loads an updated configuration to be used in the next session.
+
+```ts
+import { init } from "@eppo/react-native-sdk";
+
+await init({
+  apiKey: 'YOUR_API_KEY',
+  maxCacheAgeSeconds: 0, // Always consider cached configurations expired
+  // Settings set to non-default values
+  updateOnFetch: 'empty', // Immediately start using the new configuration once it is fetched
+  useExpiredCache: true, // Always used the previously cached assignments
+  requestTimeoutMs: 500, // Give up on fetching updated configurations after half a second and--if this is the first-ever initialization--just serve default values
+  numInitialRequestRetries: 0, // Don't retry a failed initialization fetch
+});
+```
+
+### Offline initialization
+
+Eppo's SDK supports offline initialization. This is useful if you want to initialize the SDK with a configuration from your server SDK or other external process. In this mode the SDK will not attempt to fetch a configuration from Eppo's CDN, instead only using the provided values.
+
+This function is synchronous and ready to handle assignments after it returns.
+
+```javascript
+import { offlineInit, Flag, ObfuscatedFlag } from "@eppo/react-native-sdk";
+
+// configuration from your server SDK
+const configurationJsonString: string = getConfigurationFromServer();
+
+// The configuration will be not-obfuscated from your server SDK.
+// If you have obfuscated flag values, you can use the `ObfuscatedFlag` type.
+const flagsConfiguration: Record<string, Flag | ObfuscatedFlag> = JSON.parse(configurationJsonString);
+
+offlineInit({ 
+  flagsConfiguration,
+  // If you have obfuscated flag values, you can use the `ObfuscatedFlag` type.
+  isObfuscated: true,
+ });
+```
+
+The `offlineInit` function accepts the following optional configuration arguments.
+
+### Offline Initialization Options
+
+<ApiOptionRef 
+  name="assignmentLogger"
+  type="IAssignmentLogger"
+  typeUrl="https://github.com/Eppo-exp/js-client-sdk-common/blob/75c2ea1d91101d579138d07d46fca4c6ea4aafaf/src/assignment-logger.ts#L55-L62"
+  defaultValue="null"
+>
+
+A callback that sends each assignment to your data warehouse. Required only for experiment analysis. 
+</ApiOptionRef>
+<ApiOptionRef 
+  name="flagsConfiguration"
+  type="Record <string, Flag | ObfuscatedFlag>"
+  defaultValue="null"
+>
+
+The flags configuration to use for the SDK.
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="isObfuscated"
+  type="boolean"
+  defaultValue="false"
+>
+
+Whether the flag values are obfuscated.
+</ApiOptionRef>
+<ApiOptionRef 
+  name="throwOnFailedInitialization"
+  type="boolean"
+  defaultValue="true"
+>
+
+Throw an error if an error occurs during initialization.
+</ApiOptionRef>
```

Comment:
Configuration caching and offline initialization have not yet been exposed to the React SDK. The latter is in the works in a pre-release version. We should make sure part of launching includes updating the documentation.